pkg/validation/app.go

```diff
@@ -33,6 +37,29 @@ const (
 	nameMaxLength = 53
 )
 
+var (
+	fixedProtectedNamespaces = map[string]bool{
```

Comment:
I think we should make these configurable. Leaving them as static in code will:
- bite us, when and if we change something in MAPI
- block potential opening of app platform, as it's a heavy giant swarmism

Reply:
Not sure how that would work, IMO this is a library we use all over the place so setting these in the lib which gets imported kind of makes sense to me.

Reply:
@MarcelMue exactly, in order to have it configurable we would need to do this validation outside this package, in the `app-admission-controller` directly. I talked with @piontec about it briefly, and we agreed to give it a quick try, to see how much configuration it would require, WDYT about it?